cli/lib/kontena/cli/nodes/labels/add_command.rb

```diff
@@ -2,17 +2,17 @@ module Kontena::Cli::Nodes::Labels
   class AddCommand < Kontena::Command
     include Kontena::Cli::Common
 
-    parameter "NODE_ID", "Node id"
+    parameter "NODE", "Node"
```

Comment:
Yes, it's name 99.98% of the time. We can keep the `node_id` variant as an undocumented secret :)